scripts/biomedical/diseaseOntology/unit-tests/test-output.csv

```diff
@@ -0,0 +1,25 @@
+ICD10_id,element,MESH_id,element_name,subClassOf,IAO_0000115,hasAlternativeId,hasExactSynonym,id,label,ICDO,MESH,NCI,SNOMEDCTUS20200901,UMLSCUI,ICD9CM,SNOMEDCTUS20200301,ICD10CM,SNOMEDCTUS20180301,GARD,OMIM,ORDO,EFO,MEDDRA,SNOMEDCTUS20190901
```

Comment:
This file needs to be updated to the current expected output of the script. This is currently an old version.